src/components/Homepage/jobs.js

```diff
@@ -1,68 +1,20 @@
 import React from 'react'
-import styled from 'styled-components'
-import remcalc from 'remcalc'
-import breakpoint from 'styled-components-breakpoint'
 import { Row, Col, Grid } from '../grid'
 import { Padding } from 'styled-components-spacing'
 
 import StyledLink from '../Common/StyledLink'
-import { SectionTitle, Subtitle } from '../Typography'
-import JobsByLocation from '../JobsByLocation'
-import JobLink from '../Common/JobLink'
-
-const JobContainer = styled.ul`
-  margin-top: ${remcalc(12)};
-  margin-bottom: ${remcalc(36)};
-
-  ${breakpoint('smallTablet')`
-      margin-bottom: ${remcalc(0)};
-  `}
-`
+import OpenPositions from '../Common/OpenPositions'
 
 const Jobs = () => (
   <Grid pb={3}>
+    <OpenPositions
+      data={{
+        title: 'Join our team'
+      }}
+    />
     <Row>
       <Col width={[1]}>
-        <Padding top={{ smallPhone: 3, smallTablet: 4 }}>
-          <SectionTitle>Join our team</SectionTitle>
-        </Padding>
-      </Col>
-    </Row>
-    <Row pt={[3, 3, 3, 3, 90]}>
-      <JobsByLocation>
-        {jobs =>
-          jobs.map(({ location, jobs: jobsForLocation }) => (
-            <Col
-              width={[1, 1, 1, 1, 1 / 2, 3 / 12]}
-              key={`${location}-${jobsForLocation.length}-main`}
-            >
-              <Subtitle>{location}</Subtitle>
-              <JobContainer>
-                {jobsForLocation.slice(0, 3).map((job, idx) => {
-                  const {
-                    text,
-                    hostedUrl,
-                    categories: { commitment }
-                  } = job.node
-                  return (
-                    <li key={`job-${location}-${idx}`}>
-                      <JobLink
-                        position={text}
-                        hostedUrl={hostedUrl}
-                        contractType={commitment}
-                      />
-                    </li>
-                  )
-                })}
-              </JobContainer>
-            </Col>
-          ))
-        }
-      </JobsByLocation>
-    </Row>
-    <Row>
-      <Col width={[1]}>
-        <Padding top={{ smallPhone: 3, smallTablet: 4 }}>
+        <Padding top={{ smallPhone: 3, smallTablet: 4, desktop: 3.5 }}>
           <StyledLink to="/join-us" title="View all openings">
             View all openings
```

Comment:
Following on from [this](https://github.com/yldio/yld.io/pull/277/files#r300232586) comment: 

Something else I've noticed is that even if we only have a maximum of 4 jobs per location, we're still displaying "View all openings" under the jobs list which would be weird for a user to go to view more open positions, only to find that the ones they'd seen already were the only ones we have available.

My initial suggestion was to get the number of jobs and work out if the rendered results in the homepage were capped and display a different message but the process of getting the jobs happens several levels beneath this location sooooo I would opt to just change the messaging here to be something like , "find out more about working with us", I'm not sure, speak to Hana or Greta as my copy writing skills are dire.

Reply:
We agreed with @hanalodhi that we should say `Learn more` instead of `View all openings`. 
Thank you for pointing out.